### PR TITLE
Add landing page (#13)

### LINKED ISSUE #13: Add landing page
- [ ] space for logo
- [ ] sign in button
- [ ] sign up button
- [ ] screens from app
- [ ] brief description/slogan
- [ ] footer

### PR DESCRIPTION
**Pull request by neonowy**
* Added `HomeController` for static pages.
* Added `index` action for `HomeController`.
* Added `index` view with landing page.
* Added stylesheets for landing and to customize hocus-pocus.
* Added navigation partial.
* Added footer partial.
* Added `current_context` method in `ApplicationHelper`.
* Set root route to `home#index`.
* Added font-awesome to Gemfile.
* Added controller and view tests.

Closes #13.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1 +1,17 @@
+@import "colors";
+@import "variables";
+
```
Comment (pospieszynski) [inline app/assets/stylesheets/application.scss]: empty line?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -1 +1,17 @@
...
 @import "hocus-pocus";
+
+@import "_font-awesome";
+@import "navigation";
+@import "buttons";
+@import "footer";
+@import "box";
+@import "landing";
+
```
Comment (pospieszynski) [inline app/assets/stylesheets/application.scss]: same here

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (neonowy) [inline app/assets/stylesheets/application.scss]: Imho it's more readable for me.

File: app/assets/images/browser.svg [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+                </mask>
+                <use id="Mask" fill="#FCFCFC" xlink:href="#path-1"></use>
+                <rect id="Window-Bar" fill="#DFE6EC" mask="url(#mask-2)" x="0" y="0" width="700" height="30"></rect>
+                <circle id="Oval-1" fill="#FF564F" mask="url(#mask-2)" cx="19" cy="15" r="7"></circle>
+                <circle id="Oval-1-Copy" fill="#FFB429" mask="url(#mask-2)" cx="43" cy="15" r="7"></circle>
+                <circle id="Oval-1-Copy-2" fill="#25C63A" mask="url(#mask-2)" cx="67" cy="15" r="7"></circle>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (crappylime) [inline app/assets/images/browser.svg]: no newline at end of file;)

Comment (neonowy) [inline app/assets/images/browser.svg]: This file is automatically generated by Sketch so I don't think it's really needed to add newline at the end of the file every time I export it ;)

**Comment by crappylime** [comment]
:+1:

**Comment by filip373** [comment]
:+1:

**Comment by tprochniewicz** [comment]
@gotar

**Comment by gotar** [comment]
ok you can merge

**Pull request merged**